Objects/sliceobject.c

```diff
@@ -560,6 +560,7 @@ static PyMethodDef slice_methods[] = {
      METH_O,            slice_indices_doc},
     {"__reduce__",      (PyCFunction)slice_reduce,
      METH_NOARGS,       reduce_doc},
+    {"__class_getitem__", Py_GenericAlias, METH_O|METH_CLASS, "See PEP 585"},
```

Comment:
This docstring seems misleading, as PEP 585 doesn't mention the `slice` class.

(Maybe PEP 585 could refer to https://typing.readthedocs.io/en/latest/guides/modernizing.html#generics-in-the-typing-module? Right now it links to documentation for `__class_getitem__` that on its own isn't a replacement for PEP 585 from a reader's perspective)

Reply:
Agreed it is maybe a little silly to be putting PEPs in these things when they aren't active docs.

Reply:
PRs to the PEPs repo improving things like this are welcome!

Reply:
Curious if @AlexWaygood/@JelleZijlstra have opinions on this. This is done in other places

Reply:
I do agree that our docstrings probably also shouldn't be referencing PEPs, which are, as you both note, historical documents recording decisions that were made at one point in time rather than living documentation. But all the other `__class_getitem__` docstrings say this currently, so I think it makes sense to follow suit here and change all the `__class_getitem__` docstrings to something better all at once as a followup